docs/docusaurus/versioned_sidebars/version-0.18-sidebars.json

```
@@ -225,6 +225,11 @@
           "label": "Edit a Checkpoint configuration",
           "href": "/docs/cloud/checkpoints/manage_checkpoints#edit-a-checkpoint-configuration"
         },
+        {
+          "type": "link",
+          "label": "Configure the Checkpoint result format parameter",
+          "href": "/docs/cloud/checkpoints/manage_checkpoints#configure_the_checkpoint_result_format_parameter"
```

```suggestion
          "href": "/docs/cloud/checkpoints/manage_checkpoints#configure-the-checkpoint-result-format-parameter"
```
Same as in the `docs` version of `sidebars.js`.